pyfar/classes/audio.py

```diff
@@ -56,7 +66,7 @@ def domain(self):
     def dtype(self):
         """The data type of the audio object. This can be any data type and
         precision supported by numpy."""
-        return self._dtype
+        return float if str(self._data.dtype).startswith("float") else complex
```

Comment:
What's the reason for not just returning the actual numpy dtype?

Reply:
to be consistent with the init functions. We init numpy arrays with `float` but the dtype than is `float64` or maybe `float32` depending on the system